CHANGELOG.md

```diff
@@ -0,0 +1,101 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+-
+
+## [v0.6.0-rc1] - 2018-10-16
```

Comment:
I think we may also add:

### Removed:
* **provider**: Provider field in Items got removed. Use collections instead.
* **license**: License field in Items got removed. Use collections instead.